include/odp/api/abi-default/atomic.h

```diff
@@ -57,8 +81,22 @@ struct odp_atomic_u64_s {
 	char lock; /**< Spin lock (if needed) used to ensure atomic access */
 } ODP_ALIGNED(sizeof(uint64_t)); /* Enforce alignment! */
 
+
+/**
+ * @internal
+ * Atomic 128-bit unsigned integer
+ */
+struct odp_atomic_u128_s {
+	odp_u128_t v; /**< Actual storage for the atomic variable */
+	/* Some architectures do not support lock-free operations on 128-bit
+	 * data types. We use a spin lock to ensure atomicity. */
+	char lock; /**< Spin lock (if needed) used to ensure atomic access */
+} ODP_ALIGNED(sizeof(odp_u128_t)); /* Enforce alignment! */
+
 #endif
 
+typedef struct odp_atomic_u128_s odp_atomic_u128_t;
+
```

Comment:
128 bit atomic type need to be doxygen documented in the spec file. Similarly to 32 and 64 bit atomics

```
/**
 * @typedef odp_atomic_u64_t
 * Atomic 64-bit unsigned integer
 *
 * @typedef odp_atomic_u32_t
 * Atomic 32-bit unsigned integer
 */
```

Reply:
This is done.